keras_retinanet/preprocessing/open_images.py

```diff
@@ -0,0 +1,206 @@
+import csv
```

Comment:
What is the deal with copyright header @de-vri-es ?

Reply:
There should be a copyright notice and license indication as with the other files in the repository.

@lvaleriu: Could you add them, and make sure to attribute copyright to the proper peson/company. That is most likely you, unless you made the changes during work hours for your employer (then it should be your employer).

Reply:
Done!